api/core/model_runtime/model_providers/wenxin/rerank/rerank.py

```diff
@@ -69,6 +79,7 @@ def _invoke(
             results = wenxin_rerank.rerank(model, query, docs, top_n)
 
             rerank_documents = []
+            assert "results" in results, "results key not found in response"
```

Comment:
Maybe it's better to raise a ValueError here?

Reply:
OK will change it to raise Value